sympy/functions/elementary/complexes.py

```diff
@@ -4,7 +4,7 @@
 from sympy.core.compatibility import u
 from sympy.core.exprtools import factor_terms
 from sympy.core.function import (Function, Derivative, ArgumentIndexError,
-    AppliedUndef)
+                                 AppliedUndef)
```

Comment:
Ah okay. I wasn't aware that the Python code contains rules, it looked like mere examples to me.
However, given previous statements, I think that PEP8 conformance is lower on the priority list than keeping the git log free from clutter, so I'm still -0.1. Rather the docs should be fixed to describe the factual situation: That we prefer PEP8-styled code but do not insist on following it exactly. (My personal stance is that I'm not very fanatic either way, except I don't want to see different merge officers ("Contributors") apply different standards, that's just going to frustrate contributors (coders, PR makers).)